marxs/optics/polarization.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+
+from ..math.pluecker import *
+
+
+def polarization_vectors(dir_array, angles):
+	n = len(angles)
+	polarization = np.zeros((n, 4))
+	x = np.array([1., 0., 0.])
+	y = np.array([0., 1., 0.])
+	for i in range(0, n):
+		r = h2e(dir_array[i])
+		r /= np.linalg.norm(r)
+		if not (np.isclose(r[0], 0.) and np.isclose(r[2], 0.)):
+			# polarization relative to positive y at 0              
```

Comment:
Do you think it would be easier to only switch to the x-axis convention when the direction is exactly parallel to the y-axis? Right now I am using the `np.isclose` function, but I'm not sure that's really necessary. It might not make a difference either way.